portafly/src/components/util/AlertsContext.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react'
+import {
+  AlertGroup,
+  AlertActionCloseButton,
+  Alert,
+  AlertVariant
+} from '@patternfly/react-core'
+
+interface IAlert {
+  key: string
+  variant: keyof typeof AlertVariant
+  title: string
+}
+
+const AlertsContext = React.createContext<({ addAlert?: (alert: IAlert) => void })>({})
+
+const AlertsProvider: React.FunctionComponent = ({ children }) => {
+  const [alerts, setAlerts] = React.useState<IAlert[]>([])
+
+  const removeAlert = (key: string) => {
+    setAlerts((prevAlerts) => prevAlerts.filter((a) => a.key !== key))
+  }
+
+  const addAlert = (alert: IAlert) => {
+    setAlerts((prevAlerts) => [...prevAlerts, alert])
+    setTimeout(() => removeAlert(alert.key), 8000)
```

Comment:
5000 ?

Reply:
According to @cntlsn / UX team, 8 seconds is the way to go.